framework/tlvf/tlvf_conf.yaml

```diff
@@ -10,6 +10,7 @@ include_yaml_path: {
   # "wfa_map/",
   "tlvf/wfa_map/eTlvTypeMap.yaml",
   "tlvf/wfa_map/tlvApCapability.yaml",
+  "tlvf/wfa_map/tlvApMetricQuery.yaml",
```

Comment:
This, change, the change of yaml file, and the auto-generated files must be in a separate commit (previous to this commit)

Reply:
noted for next time if i may.

Reply:
No, please fix it in this PR.